Mage.Sets/src/mage/cards/m/MeteorSwarm.java

```diff
@@ -19,10 +19,11 @@ public MeteorSwarm(UUID ownerId, CardSetInfo setInfo) {
 
         // Meteor Swarm deals 8 damage divided as you choose among X target creatures and/or planeswalkers.
         this.getSpellAbility().addEffect(
-                new DamageMultiEffect(8).
+                new DamageMultiEffect().
                         setText("{this} deals 8 damage divided as you choose among X target creatures and/or planeswalkers.")
         );
-        this.getSpellAbility().addTarget(new TargetCreatureOrPlaneswalkerAmount(8));
+        // Minimum number of targets will be overridden to X by the adjuster
+        this.getSpellAbility().addTarget(new TargetCreatureOrPlaneswalkerAmount(8, 1));
```

Comment:
can you explain why this isn't min targets 0?

Reply:
Meteor Swarm uses a target adjuster to make sure that you pick exactly X targets. The `XTargetsCountAdjuster` it uses is built on `TargetsCountAdjuster`, which will not change the minNumberOfTargets if the min is already set to zero.